lib/src/util/react_wrappers.dart

```diff
@@ -195,7 +195,7 @@ Map getProps(dynamic/* ReactElement|ReactComponent */ instance, {bool traverseWr
 ///
 /// This method simply wraps react.findDOMNode with strong typing for the return value
 /// (and for the function itself, which is declared using `var` in react-dart).
-Element? findDomNode(dynamic instance) => react_dom.findDOMNode(instance) as Element?;
+Element? findDomNode(ReactNode instance) => react_dom.findDOMNode(instance) as Element?;
```

Comment:
I don't think `ReactNode` is the appropriate type here, since it should be the mounted component instance (`ReactComponent|react.Component|Element`), not the VDOM instance. So, I think we should switch this back to `dynamic`.